docs/source/fr/installation.md

```diff
@@ -0,0 +1,265 @@
+```
+
+## Installation modifiable
+
+Vous aurez besoin d'une installation modifiable si vous le souhaitez :
+
+  * Utiliser la version de la branche `main` du code source.
+  * Contribuer à 🤗 Transformers et vouler tester vos modifications du code source.
+
+Clonez le projet et installez 🤗 Transformers avec les commandes suivantes :
+
+```bash
+git clone https://github.com/huggingface/transformers.git
+cd transformers
+pip install -e .
+```
+
+Ces commandes créent des liens entre le dossier où le projet a été cloné et les chemins de vos librairies Python.
+Python regardera maintenant dans le dossier que vous avez cloné en plus des chemins de vos librairies normaux.
+Par exemple, si vos librairies Python sont installées dans `~/anaconda3/envs/main/lib/python3.7/site-packages/`, Python cherchera aussi dans le dossier où vous avez cloné : `~/transformers/`.
+
+<Tip warning={true}>
+
+Vous devez garder le dossier `transformers` si vous voulez continuer d'utiliser la librairie.
+
+</Tip>
+
+Maintenant, vous pouvez facilement mettre à jour votre clone avec la dernière version de 🤗 Transformers en utilisant la commande suivante :
+
+```bash
+cd ~/transformers/
+git pull
+```
+
+Votre environnement Python trouvera la version de la branche `main` lors de la prochaine exécution.
+
+## Installation avec conda
+
+Installation avec la chaîne `huggingface` de conda :
+
+```bash
+conda install -c huggingface transformers
+```
+
+## Configuration du cache
+
+Les modèles pré-entraînés sont téléchargés et cachés localement dans le dossier suivant : `~/.cache/huggingface/hub`. C'est le dossier par défaut donné par la variable d'environnement `TRANSFORMERS_CACHE`. Sur Windows, le dossier par défaut est `C:\Users\nom_utilisateur\.cache\huggingface\hub`. Vous pouvez modifier les variables d'environnement indiquées ci-dessous - par ordre de priorité - pour spécifier un dossier de cache différent :
+
+1. Variable d'environnement (par défaut) : `HUGGINGFACE_HUB_CACHE` ou `TRANSFORMERS_CACHE`.
+2. Variable d'environnement : `HF_HOME`.
+3. Variable d'environnement : `XDG_CACHE_HOME` + `/huggingface`.
+
+<Tip>
+
+🤗 Transformers utilisera les variables d'environnement `PYTORCH_TRANSFORMERS_CACHE` ou `PYTORCH_PRETRAINED_BERT_CACHE` si vous utilisez une version précédente de cette librairie et avez défini ces variables d'environnement, sauf si vous spécifiez la variable d'environnement `TRANSFORMERS_CACHE`.
+
+</Tip>
+
+## Mode hors ligne
+
+🤗 Transformers peut fonctionner dans un environnement cloisonné ou hors ligne en n'utilisant que des fichiers locaux. Définissez la variable d'environnement `TRANSFORMERS_OFFLINE=1` pour activer ce mode.
+
+<Tip>
+
+Ajoutez [🤗 Datasets](https://huggingface.co/docs/datasets/) à votre processus d'entraînement hors ligne en définissant la variable d'environnement `HF_DATASETS_OFFLINE=1`.
+
+</Tip>
+
+Par example, vous exécuteriez généralement un programme sur un réseau normal cloisonné des instances externes avec la commande suivante :
```

Comment:
complicated way of saying that you are running something under proxy

Reply:
It appears that is particular sentence was removed from the English documentation. I updated the content to match the English documentation. I am a bit unsure about the translation of the last sentence (i.e., `Seuls les fichiers ...`), please check.